SpatialGDK/Documentation/content/get-started/build-unreal-fork.md

```diff
@@ -1,50 +1,66 @@
 
 1. Go to **Control Panel > System and Security > System > Advanced system settings > Advanced > Environment variables**.
-1. Create a system variable named **UNREAL_HOME**.
-1. Set the variable value to be the path to the directory you cloned the Unreal Engine fork into.
-1. Make sure that the new environment variable is registered by restarting your terminal and running `echo %UNREAL_HOME%` (Command Prompt) or `echo $Env:UNREAL_HOME` (PowerShell). If the environment variable is registered correctly, this returns the path to the directory you cloned the Unreal Engine fork into. If it doesn’t, check that you’ve set the environment variable correctly.
-1. Create a system variable named **LINUX_MULTIARCH_ROOT**.
-1. Set the variable value to be the path to the directory of your unzipped Linux cross compilation toolchain.
-1. Make sure that the new environment variable is registered by restarting your terminal and running `echo %LINUX_MULTIARCH_ROOT%` (Command Prompt) or `echo $Env:LINUX_MULTIARCH_ROOT` (PowerShell).
-If the environment variable is registered correctly, this returns the path you unzipped `v11_clang-5.0.0-centos7.zip` into. If it doesn’t, check that you’ve set the environment variable correctly.
-
-### Building Unreal Engine
-
-> Building the Unreal Engine fork from source could take up to a few hours.
-
-1. Open **File Explorer** and navigate to the directory you cloned the Unreal Engine fork into.
-1. Double-click **`Setup.bat`**.
-</br>This installs prerequisites for building Unreal Engine 4, and may take a while.
-    >  While running the Setup file, you should see `Checking dependencies (excluding Mac, Android)...`. If it also says `excluding Linux`, make sure that you set the environment variable `LINUX_MULTIARCH_ROOT` correctly, and run the Setup file again.
+2. Create a system variable named **UNREAL_HOME**.
+3. Set the variable value to the path to the directory you cloned the Unreal Engine fork into.
+4. Restart your terminal and run `echo %UNREAL_HOME%` (Command Prompt) or `echo$Env:UNREAL_HOME` (PowerShell). If you have registered the environment variable correctly, this returns the path to the directory you cloned the Unreal Engine fork into. If it doesn’t, check that you’ve set the environment variable correctly.
+5. Create a system variable named **LINUX_MULTIARCH_ROOT**.
+6. Set the variable value to the path to the directory of your unzipped Linux cross compilation toolchain.
+7. Restart your terminal and run `echo %LINUX_MULTIARCH_ROOT%` (Command Prompt) or `echo $Env:LINUX_MULTIARCH_ROOT` (PowerShell). If you have registered the environment variable correctly, this returns the path you unzipped `v11_clang-5.0.0-centos7.zip` into. If it doesn’t, check that you’ve set the environment variable correctly.
+
+### Step 4: Building Unreal Engine
```

Comment:
Just realised we are gerund-tastic (building, adding - lots of ings) here - wondering why we did this, when we are trying to be all imperative (build, add).

Reply:
Did an "ing" purge. See the latest commit.